src/pages/ReportSettingsPage.js

```diff
@@ -65,6 +66,21 @@ class ReportSettingsPage extends Component {
         ];
     }
 
+    /**
+     * @param {Object} report - the given report we're viewing settings for
+     * @param {Object|null} policy - the workspace the report is on, null if the user isn't a member of the workspace
+     * @returns {Boolean}
+     */
+    getShouldDisablePublicRoomRename(report, policy) {
```

Comment:
```suggestion
    shouldDisablePublicRoomRename(report, policy) {
```

Reply:
Agreed, the naming is not according to guidelines.